.github/workflows/update-babel-types-docs.yml

```diff
@@ -0,0 +1,55 @@
+name: Update Babel types docs
+env:
+  YARN_ENABLE_SCRIPTS: false # disable post-install scripts
+on:
+  workflow_dispatch:
+    inputs: {}
+  schedule:
+    - cron: "0 0 * * 5"
```

Comment:
Can we somehow dispatch this job when adding a commit in the Babel repository? Maybe with webhooks?

Reply:
Or we can move this workflow to the main repo, as long as the bot can open a PR to the website repo.